Author: Adam Gausmann <adam@gaussian.dev>
Date:   Sun Dec 1 18:20:33 2024 -0600

Remove calls to pthread_getname_np.

Void is currently on musl version 1.1, and musl added pthread_getname_np
in version 1.2.

This only affects some debug logs, which will fallback to the numeric
thread ID instead of the thread name.

--- a/src/xrt/auxiliary/util/u_linux.c
+++ b/src/xrt/auxiliary/util/u_linux.c
@@ -53,10 +53,6 @@ get_name(char *str, size_t count)
 	// First init.
 	str[0] = '\0';
 
-	// Get name of thread.
-	pthread_t this_thread = pthread_self();
-	pthread_getname_np(this_thread, str, count);
-
 	if (str[0] == '\0') {
 		snprintf(str, count, "tid(%i)", gettid());
 	}
